File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int maxP = 0;
        for(int i=0;i<prices.size()-1;i++){
            for(int j=i+1;j<prices.size();j++){
               int p = prices[j]-prices[i];
               maxP = max(maxP,p);
            }
        }
        return maxP;
    }
};

File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] heights) {
        int n = heights.length;
        int max_area = 0;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){// fix i and loop for j
            max_area = Math.max(max_area,Math.min(heights[i],heights[j])*(j-i));
            // always get min height container bcz container with max height 
            // water flows out of it so no use 
            }
        }
        return max_area;
    }
}
